common/scripts/metallb.yaml

```diff
@@ -307,45 +358,63 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: status.podIP
+            # needed when another software is also using memberlist / port 7946
+            # when changing this default you also need to update the container ports definition
+            # and the PodSecurityPolicy hostPorts definition
+            #- name: METALLB_ML_BIND_PORT
+            #  value: "7946"
             - name: METALLB_ML_LABELS
               value: "app=metallb,component=speaker"
-            - name: METALLB_ML_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
             - name: METALLB_ML_SECRET_KEY
               valueFrom:
                 secretKeyRef:
                   name: memberlist
                   key: secretkey
-          image: metallb/speaker:v0.9.3
-          imagePullPolicy: Always
+          image: quay.io/metallb/speaker:v0.12.1
           name: speaker
           ports:
             - containerPort: 7472
               name: monitoring
-          resources:
-            limits:
-              cpu: 100m
-              memory: 100Mi
+            - containerPort: 7946
+              name: memberlist-tcp
+            - containerPort: 7946
+              name: memberlist-udp
+              protocol: UDP
+          livenessProbe:
+            httpGet:
+              path: /metrics
+              port: monitoring
+            initialDelaySeconds: 10
```

Comment:
do we really need this so long? This may make it take 20s to get ready, slowing down everything